### ISSUE TITLE
Create a vanity import path at "nmstate.io/nmpolicy"

### ISSUE THREAD

**Issue by qinqon** [nmstate/nmpolicy#53]
Following the path like other projects like kubevirt or k8s are consumed by other projects, maybe nmpolicy that is going to be consumed by kubernetes-nmstate and other projects need a vanity import like `nmstate.io/nmpolicy` that points to `github.com/nmstate/nmpolicy`, so the project is not stick to github.

One possible implementation is modify the "nmstate.io" page header and add the following meta:

```html
<meta name="go-import" content="nmstate.io/nmpolicy git https://github.com/nmstate/nmpolicy">
```

This is how it's done at kubevirt: https://github.com/kubevirt/kubevirt.github.io/blob/main/_includes/head.html#L6-L18

Another alternative is to use a subdomain like `go.nmstate.io` similart to `go.uber.org` or `go.opencensus.io` so we don't have to modify the main webpage at `nmstate.io`.

**Comment by EdDev** [comment]
If I understand this right, the motivation is to avoid being dependent on github.com?
Lets assume that a decision is taken to use some other source control hosting platform, can't we just do it then and declare a new version?

**Comment by EdDev** [comment]
I must say, that I do not mind or have any hard opinion on this. I just want to "warn" about the domain, which is actually owned by Red Hat and not by any community organization.
In fact, most domains will be owned by someone, making that the dependency instead of github, so I am not sure if that is any better.

**Comment by qinqon** [comment]
Right, same happends with `kubevirt.io`, I suppose it's owner by RedHat, but my main concern is that if we change to vanity imports when this project start to have some users (it's not like is going to be huge) we may end up with issues, but I think they are not that bad.
